src/Effects.js

```diff
@@ -485,6 +489,148 @@ module.exports = class Effects {
     this.ripples = createRipplesEffect(false);
     this.ripples_random = createRipplesEffect(true);
 
+    this.blooming_petals = {
```

Comment:
Maybe worth a comment on each of these new ones mentioning where they came from?